charts/hub-agent/templates/deployment.yaml

```diff
@@ -53,6 +53,10 @@ spec:
             - --cluster-unhealthy-threshold={{ .Values.clusterUnhealthyThreshold }}
             - --resource-snapshot-creation-minimum-interval={{ .Values.resourceSnapshotCreationMinimumInterval }}
             - --resource-changes-collection-duration={{ .Values.resourceChangesCollectionDuration }}
+            - --azure-property-checker-enabled={{ .Values.azurePropertyChecker.isEnabled }}
```

Comment:
i'm thinking of not updating these charts at all to avoid the backport conflicts, as these default values of flags should be good. I don' think we will add e2e test in this repo.